src/app/src/test/kotlin/org/vaccineimpact/api/tests/controllers/ResponsibilityControllerTests.kt

```diff
@@ -110,8 +110,10 @@ class ResponsibilityControllerTests : MontaguTests()
 
 
     private val mockTouchstones = listOf(
-            TouchstoneVersion("touchstone-1", "touchstone", 1, "Description", TouchstoneStatus.OPEN),
-            TouchstoneVersion("touchstone-bad", "touchstone", 1, "not open", TouchstoneStatus.IN_PREPARATION)
+            Touchstone("touchstone", "description", "comment", listOf(
```

Comment:
We also need to update the test that we're only returning open touchstone versions, `does not return in preparation touchstones if user has no permission to read prepared touchstones`